CUDADataFormats/HGCal/interface/HGCRecHitSoA.h

```diff
@@ -0,0 +1,27 @@
+#ifndef CUDADataFormats_HGCal_HGCRecHitSoA_h
+#define CUDADataFormats_HGCal_HGCRecHitSoA_h
+
```

Comment:
Thanks for pointing this out. In addition, I took the liberty to replace all ```uint32_t``` by ```std::uint32_t``` (same for other types) to avoid any possible ambiguity.